osu.Game/Overlays/Settings/Sections/Gameplay/GeneralSettings.cs

```diff
@@ -63,6 +64,12 @@ private void load(OsuConfigManager config)
                     LabelText = "Always show key overlay",
                     Current = config.GetBindable<bool>(OsuSetting.KeyOverlay)
                 },
+                new SettingsSlider<float, TimeSlider>
```

Comment:
Not sure I agree with this one being in here... With that being said I also don't agree with the gameplay one being tied to the game-wide/exit one, but that's something I've argued in the past.

Reply:
It's not gameplay specific so agree it shouldn't be here (this also affects the main menu exit timer, for instance).